wordpress-mobile/WordPress-iOS
# Zendesk Mobile: Me tab: update icon for new notifications

Ref #9325 

To test:
- Start the app.
- Go to Support and create a ticket.
- Be on a tab other than Me.
- Reply to your ticket in Zendesk.
- When the PN is received:
  - Verify the Me icon changes to unread.
  - Select the Me tab. Verify the Me icon is now blue (still unread).
- Select My Tickets to clear the notification.
  - Verify the Me icon is now the original.
  - Select another tab. Verify the Me icon is still the original.

Unread notifications icon:
<img width="400" alt="unselected_unread" src="https://user-images.githubusercontent.com/1816888/40087382-7c0c7a4c-585f-11e8-9dcc-6fec15fb85ba.png">
<img width="400" alt="selected_unread" src="https://user-images.githubusercontent.com/1816888/40191414-bbf1d854-59be-11e8-8a4a-fcca2b45ea49.png">




No notifications icon: (no change, just verification)
<img width="400" alt="unselected_read" src="https://user-images.githubusercontent.com/1816888/40087442-bf41cc86-585f-11e8-84cd-4cdd4f24b01b.png">
<img width="400" alt="selected_read" src="https://user-images.githubusercontent.com/1816888/40087452-c8e3bf10-585f-11e8-8bc0-a8837c330460.png">
